product-service/serverless.yaml

```diff
@@ -45,4 +62,32 @@ functions:
           request:
             parameters:
               paths:
-                productId: true
\ No newline at end of file
+                productId: true
+
+  catalogBatchProcess:
+    handler: handler.catalogBatchProcess
+    events:
+      - sqs:
+          batchSize: 5
+          arn:
+            Fn::GetAtt:
+              - SQSQueue
+              - Arn
+
+resources:
+  Resources:
+    SQSQueue:
+      Type: AWS::SQS::Queue
+      Properties:
+        QueueName: catalogItemsQueue
+    SNSTopic:
+      Type: AWS::SNS::Topic
+      Properties:
+        TopicName: createProductTopic
+    SNSSubscription:
+      Type: AWS::SNS::Subscription
+      Properties:
+        Endpoint: constantlearning1985@gmail.com
```

Comment:
it would be better to storage such data in env file with usage of serverless-dotenv-plugin